core/src/main/java/tachyon/master/MasterInfo.java

```diff
@@ -670,6 +678,8 @@ boolean _delete(int fileId, boolean recursive, long opTimeMs) throws TachyonExce
         delInode.reverseId();
       }
 
+      mMasterSource.incFilesDeleted(delInodes.size());
+      mMasterSource.incDeleteFileOps();
```

Comment:
I'm still not too clear on when the `inc*Ops` counter should be incremented. If the counter represents the number of successful method calls, maybe we should prefix it like `incSuccessful*Ops`? Also for `delete`, should it be incremented when the inode is null, since we return a successful response (true).